components/tor/tor_switches.h

```diff
@@ -0,0 +1,15 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_TOR_TOR_SWITCHES_H_
+#define BRAVE_COMPONENTS_TOR_TOR_SWITCHES_H_
+
+namespace tor {
+
+extern const char kDisableTorClientUpdaterExtension[];
```

Comment:
How about using `constexpr` instead of having separated definitions and declaration.
It would be good for compile time optimization.

Reply:
addressed in e1726d757df0524a90d1c7a961847bebc7ed8eee